src/averageAges.js

```diff
@@ -15,11 +15,14 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const filterMen = !century
+    ? people.filter((person) => person.sex === 'm')
+    : people.filter((person) => {
+      return person.sex === 'm' && Math.ceil(person.died / 100) === century;
+    });
+
+  return filterMen.reduce((sum, person) => (
+    sum + (person.died - person.born)), 0) / filterMen.length;
```

Comment:
```suggestion
  return filterMen.reduce((sum, { died, born }) => (
    sum + (died - born)), 0) / filterMen.length;
```

Here and across your code you can find places where you can destructure objects, so you won't need to use dot notation.